cli/install/install.go

```diff
@@ -1061,6 +1074,13 @@ func Install(binDir string, includeDir string, installCtx InstallCtx,
 	local string, cliOpts *config.CliOpts) error {
 	var err error
 
+	// This check is needed for knowing that we will be able to copy
+	// recently built binaries to the corresponding bin and include directories.
+	if !dirsAreWriteable([]string{binDir, includeDir}) {
+		return fmt.Errorf("some of directories are not writeable for the current user. " +
+			"Please, use sudo for the successful install")
```

Comment:
You don't have to use `sudo`, you can just update rights (the user have a choice)?

Reply:
Fixed the output, thank you.